src/mcts/search.cc

```diff
@@ -40,9 +40,9 @@
 #include "mcts/node.h"
 #include "neural/cache.h"
 #include "neural/encoder.h"
-#include "utils/spinhelper.h"
 #include "utils/fastmath.h"
 #include "utils/random.h"
+#include "utils/spinhelper.h"
```

Comment:
Just checking that this is intentional here, as it isn't related to contempt.

Reply:
There were some minor style issues in #1890 that I didn't notice when reviewing, but clang-format fixed.